src/lm_buddy/jobs/configs/base.py

```diff
@@ -16,6 +20,15 @@ class LMBuddyJobConfig(BaseLMBuddyConfig):
     but this is not rigidly constrained by the interface. This may change in the future.
     """
 
+    name: str = Field(description="Name of the job.")
```

Comment:
Making both of these inherited fields on `LMBuddyJobConfig`, since every job config should have them.